consensus/XDPoS/XDPoS.go

```diff
@@ -208,21 +182,12 @@ func (x *XDPoS) VerifyHeaders(chain consensus.ChainReader, headers []*types.Head
 	results := make(chan error, len(headers))
 
 	// Split the headers list into v1 and v2 buckets
```

Comment:
nit: "Split the headers list into v1 and v2 buckets" can be removed

Reply:
👍